### DISCUSSION TITLE
Looping Restarts after upgrade

### DISCUSSION THREAD

**Discussion by GabeShack** [grahampugh/erase-install#452] [Q&A]
Hey all,
Just wanted to check if anyone was seeing looping restarts after attempting upgrade from 10.15-13.6  

We use firmware passwords and I figured this was part of the issue, so ive since removed them, however i have about 5-10 that attempted the update themselves, and now are caught in this rebooting loop.  Ive reinstalled 13.6 on them from both the recovery and from a boot usb drive as well as internet recovery...with the same result.  Im attempting to now try to move one to sonoma to see if that resolves it.

Ive done PRAM resets...pmu resets, i did a revive from configurator.  Ive done everything except wipe them fully...which i guess is the next step.

Ive also done safe booting with shift held down.  Not sure what the cause is, but it's really annoying.

**Comment by meanwhenhungry** [comment]
Your issue and answer is in the Requirements

**macOS 11** is required. If you need to run erase-install.sh on older versions of macOS, use version 27.x.
The file system must be APFS.
